### PR TITLE
Adapt e2e test to test with operator

### PR DESCRIPTION
**Pull request by mandydydy**
- update submodule with new attractor format
- adding back multi-trench test
- add manifests for deploying meridio from operator

### CODE REVIEW

File: test/e2e/e2e.md [hunk 1]
```diff
@@ -8,56 +8,56 @@
...
-    # this step(step 4) needs to be done for multiple times depending on how many trenches are deployed
-    # the <meridio-service-account-name>, for example, for trench-a, is "meridio-trench-a" if helm chart is used, "meridio-sa-trench-a" if operator is used
-    ./docs/demo/scripts/spire.sh <meridio-service-account-name> $ns
-    ```
+   ```bash
+   ns=<testing-namespace>
+   ./docs/demo/scripts/spire.sh default $ns
+   # this step(step 4) needs to be done for multiple times depending on how many trenches are deployed
+   # the <meridio-service-account-name>, for example, for trench-a, is "meridio-trench-a" if helm chart is used, "meridio-sa-trench-a" if operator is used
+   ./docs/demo/scripts/spire.sh <meridio-service-account-name> $ns
```
Comment (LionelJouin) [inline test/e2e/e2e.md:34]: can be moved to step. 9 with corresponding service account

File: test/e2e/e2e.md [hunk 2]
```diff
@@ -8,56 +8,56 @@
...
 
 7. Install targets connected to trench-a
 
-    ```bash
-    # <conduit-name> is load-balancer if testing with helm, and lb-fe if testing with the operator
-    helm install examples/target/helm/ --generate-name --create-namespace --namespace $ns --set applicationName=target-a --set default.trench.name=trench-a --set default.conduit.name=<conduit-name>
-    ```
+   ```bash
+   # <conduit-name> is load-balancer if testing with helm, and lb-fe if testing with the operator
+   helm install examples/target/helm/ --generate-name --create-namespace --namespace $ns --set applicationName=target-a --set default.trench.name=trench-a --set default.conduit.name=<conduit-name>
```
Comment (LionelJouin) [inline test/e2e/e2e.md:53]: can be moved to step. 9 with corresponding conduit name

File: test/e2e/e2e.md [hunk 3]
```diff
@@ -67,19 +67,19 @@ Meridio should test with the operator before each release to ensure the compatib
...
+   # update the Meridio Operator submodule
+   git pull
+   git submodule sync --recursive
+   git submodule update --init --recursive
+   # deploy certificate manager
+   kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
+   # run at the git root directory
+   cd Meridio-Operator
+   # make sure the operator image is on the cluster or can be pulled by the cluster
+   make deploy NAMESPACE=$ns
```
Comment (LionelJouin) [inline test/e2e/e2e.md:80]: The image might be required, I tried it and I get: "Failed to pull image "controller:0.0.1""

**Review by LionelJouin — APPROVED** [review]

File: test/e2e/e2e.md (hunk 3 above)
Comment (mandydydy) [reply test/e2e/e2e.md:80]: the comment in line 79 says "make sure the operator image is ..." 
maybe I should make it an explicit step

**Review by estinfra — COMMENTED** [review]
FOSS Introduction is done. Please check the [logs](https://artifactory.est.tech/artifactory/proj-est-nsm-generic-local/reports/meridio/master/b979650).

**Pull request merged**